parsl/utils.py

```diff
@@ -24,3 +24,37 @@ def get_version():
         logger.exception("Unable to determine code state")
 
     return version
+
+def get_last_checkpoint():
+  """Finds the checkpoint from the last run, if one exists.
+
+  Note that checkpoints are incremental, and this helper will not find
+  previous checkpoints from earlier than the most recent run. It probably
```

Comment:
I think we need to clarify that this will only find checkpoints that happen to be in `rundir`, unless we switch to reading off `DataFlowKernel.rundir`, in which case we should clarify that this will only find checkpoints that use the same rundir as the current script.

Also we may want to allow one to specify `n` last checkpoints, rather than just the most recent?